spot-client/src/spot-tv/app-state/actions.js

```diff
@@ -154,6 +155,7 @@ export function createSpotTVRemoteControlConnection({ pairingCode, retry }) {
             dispatch(setTenant(tenant));
             dispatch(setPermanentPairingCode(permanentPairingCode));
             roomProfile && dispatch(setCustomerId(roomProfile.customerId));
+            dispatch(reconnectScheduleUpdate(false));
```

Comment:
Similar to remote, the TV also has explicit disconnect method which duplicates the cleanup in `disconnectSpotTvRemoteControl`. I would put this call for safety there as well. Or maybe test that scenario which might not be easy or maybe we dont do explicit disconnect yet while reconnecting anywhere in the app... so maybe just better put it there?